logging/src/test/kotlin/com/hedvig/libs/logging/masking/MaskedTest.kt

```diff
@@ -250,4 +252,21 @@ class MaskedTest {
         val d = BigDecimal(12)
         assertThat(d.toMaskedString()).isEqualTo(d.toString())
     }
+
+    @Test
+    fun testResponseEntity() {
+        data class Child(
+            val a: String,
+            val b: String,
+            @Masked val c: String,
+        )
+
+        val a = Child("a", "2", "masked")
+        val b = ResponseEntity<Child>(a, HttpStatus.OK)
```

Comment:
since we're using a, b and c above
```suggestion
        val child = Child("a", "2", "masked")
        val responseEntity = ResponseEntity<Child>(child, HttpStatus.OK)
```

Reply:
Fixed!